miden-tx/src/lib.rs

```diff
@@ -40,9 +40,15 @@ pub use error::{
 
 pub mod auth;
 
+#[cfg(feature = "testing")]
+pub mod testing;
+
 #[cfg(test)]
 mod tests;
 
+#[cfg(all(test, feature = "testing"))]
+mod kernel_tests;
```

Comment:
Should `test` always imply `testing`?

Reply:
Yes, I had modified the toml but did not update the guard. Removed.